src/chum.cpp

```diff
@@ -99,14 +99,18 @@ void Chum::refreshPackages() {
   m_packages_last_refresh.clear();
   m_packages_last_refresh_installed.clear();
 
-  auto pktr = Daemon::getPackages(Transaction::FilterNotDevel);
+  auto pktr = Daemon::getPackages(Transaction::FilterNotSource);
   //% "Get list of packages"
   setStatus(qtTrId("chum-get-list-packages"));
   connect(pktr, &Transaction::package,  this, [this](
           [[maybe_unused]] int info,
           const QString &packageID,
           [[maybe_unused]] const QString &summary
           ) {
+    QString pn = Daemon::packageName(packageID);
+    if (pn.endsWith(QStringLiteral("-debuginfo")) ||
```

Comment:
On the of the clazy checks suggests using QLatin1String in cases like this.

Reply:
Will swap for those. Thanks